Cargo.toml

```diff
@@ -45,15 +45,42 @@ ptr_as_ptr = "warn"
 ptr_cast_constness = "warn"
 ref_as_ptr = "warn"
 
+std_instead_of_core = "warn"
+std_instead_of_alloc = "warn"
+alloc_instead_of_core = "warn"
+
 [workspace.lints.rust]
 missing_docs = "warn"
 unexpected_cfgs = { level = "warn", check-cfg = ['cfg(docsrs_dep)'] }
 unsafe_code = "deny"
 unsafe_op_in_unsafe_fn = "warn"
 unused_qualifications = "warn"
 
-[lints]
-workspace = true
```

Comment:
We explicitly _don't_ want examples to be subject to the same linting requirements as the rest of Bevy, since it is encouraged that users rely on the Rust `std` library when possible. Unfortunately, there is no way to override just those lints, so a copy of the workspace lints is needed.

Reply:
Please add a comment to this effect here and in the workspace lints to reduce the risk of desync.

Reply:
Done!